### PR TITLE
add chinese link in gen_doc.py

### PR DESCRIPTION
**Pull request by tink2123**

### CODE REVIEW

File: doc/fluid/api/gen_doc.py [hunk 1]
```diff
@@ -109,6 +111,9 @@ def _print_header_(self, name, dot, is_title):
     def _print_ref_(self, name):
         self.stream.write(".. _api_{0}_{1}:\n\n".format("_".join(
             self.module_name.split(".")), name))
+    def _print_cn_ref(self,name):
+    	self.stream.write("Read Chinese Version: :ref:`cn_api_{0}_{1}`\n\n".format("_".join(
```
Comment (shanyi15) [inline doc/fluid/api/gen_doc.py]: 这里缩进是否有问题呢？

File: doc/fluid/api/gen_doc.py [hunk 2]
```diff
@@ -109,6 +111,9 @@ def _print_header_(self, name, dot, is_title):
     def _print_ref_(self, name):
         self.stream.write(".. _api_{0}_{1}:\n\n".format("_".join(
             self.module_name.split(".")), name))
+    def _print_cn_ref(self,name):
```
Comment (shanyi15) [inline doc/fluid/api/gen_doc.py]:
```suggestion
    def _print_cn_ref(self, name):
```

**Review by shanyi15 — COMMENTED** [review]
Nice Job! Just some format polishment.

**Comment by tink2123** [comment]
@shanyi15 fixed！thanks very much~

**Review by shanyi15 — APPROVED** [review]
LGTM

**Pull request merged**